function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
```

Comment:
If we can rename the function to its' purpose will help us for future reference.